htdocs/core/tpl/card_presend.tpl.php

```diff
@@ -209,7 +209,7 @@
 	}
 
 	$formmail->withto = $liste;
-	$formmail->withtofree = (GETPOSTISSET('sendto') ? (GETPOST('sendto') ? GETPOST('sendto') : '1') : '1');
+	$formmail->withtofree = (GETPOSTISSET('sendto') ? (GETPOST('sendto', 'alpha') ? GETPOST('sendto', 'alpha') : '1') : '1');
```

Comment:
In v12, no param is same than 'alpha'. So PR seems useless here.

Reply:
No in v12 it's alphanohtml by default
https://github.com/Dolibarr/dolibarr/blob/440f923b014c9131610baa145784334042118459/htdocs/core/lib/functions.lib.php#L302
and case alpha is different than case alphanohtml in v12 : 
https://github.com/Dolibarr/dolibarr/blob/440f923b014c9131610baa145784334042118459/htdocs/core/lib/functions.lib.php#L560
https://github.com/Dolibarr/dolibarr/blob/440f923b014c9131610baa145784334042118459/htdocs/core/lib/functions.lib.php#L598